Sources/ARKit-CoreLocation/Nodes/LocationNode.swift

```diff
@@ -156,6 +156,11 @@ open class LocationNode: SCNNode {
         SCNTransaction.begin()
         SCNTransaction.animationDuration = setup ? 0.0 : 0.1
 
+        let distance = self.location(locationManager.bestLocationEstimate).distance(from:
+            locationManager.currentLocation ?? nodeLocation)
+
+        childNodes.first?.renderingOrder = Int.max - 1000 - (Int(distance * 1000))
```

Comment:
can we DRY up the `Int.max - 1000 - (Int(distance * 1000))`?  It's used in both `LocationAnnotationNode` and `LocationNode`.  We could turn this into a function in `LocationNode` and then it would be available in both classes.

Reply:
Great idea.  Done.  And with comments to explain the (necessary but seemingly out of place) constant.